docs/b2b-edition/specs/storefront/storefront/company.yaml

```diff
@@ -96,3932 +86,817 @@ paths:
               schema:
-                description: ''
-                type: object
-                properties:
-                  code:
-                    type: number
-                  message:
-                    type: string
-                    minLength: 1
-                  data:
-                    type: object
+                allOf:
+                  - $ref: "#/components/schemas/responseObject"
+                  - type: object
                     properties:
-                      userId:
+                      code:
                         type: number
-                    required:
-                      - userId
-                required:
-                  - code
-                  - message
-                  - data
-                x-examples:
-                  example-1:
-                    code: 200
-                    message: SUCCESS
-                    data:
-                      userId: 7838
-              examples:
-                example-1:
-                  value:
-                    code: 200
-                    message: SUCCESS
-                    data:
-                      userId: 7838
+                        default: 200
+                      data:
+                        type: object
+                        properties:
+                          list:
+                            type: array
+                            items:
+                              allOf:
+                                - $ref: "#/components/schemas/userListItems_BASE"
+                                - type: object
+                                  properties:
+                                    uuid:
+                                      type: string
+                                      description: "A unique ID associated with the user's account. Does not match the `id` field by default."
+                                      format: uuid
+                                    catalogId:
+                                      $ref: "#/components/schemas/catalogId"
+                          pagination:
+                            $ref: "#/components/schemas/responsePagination"
+    post:
+      tags:
+        - Companies
+        - Users
+      summary: "Create a Company User"
+      operationId: post-companies-companyId-users
       description: |-
-        Create company user.
+        Creates a user associated with a particular Company account.
+
         Equivalent Storefront GraphQL API Mutation: `userCreate`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
+
+        If you enter an email address in the `email` field that is associated with a BigCommerce customer account, and the customer account does not already have a corresponding Company user account, it will link the existing customer account with the new Company user. If the email is not associated with a customer account, a new one will be created with the Company user.
```

Comment:
In the main intro content, I thought it was potentially vague to refer to B2B user accounts vs "storefront customer accounts."  But I thought maybe that was prescribed terminology.  Here, though, we do explicitly refer to "BigCommerce customer account."  I would think that would be more explicit and clear to use in the intro as well.